compiler/src/iree/compiler/Bindings/Native/Transforms/test/wrap_entry_points.mlir

```diff
@@ -229,3 +229,88 @@ util.func private @importEncodingsCaller(%arg0: tensor<?x2xi32>) -> tensor<2x?xi
   %0 = util.call @importEncodings(%arg0) : (tensor<?x2xi32>) -> tensor<2x?xi32>
   util.return %0 : tensor<2x?xi32>
 }
+
+// -----
+
+// Tests that iree.abi.transients attribute wraps results with hal.tensor.transients.
+
+// CHECK-LABEL: util.func public @transientsEntry(
+//  CHECK-SAME:   %[[ARG0:.+]]: !hal.buffer_view, %[[STORAGE:.+]]: !hal.buffer) -> !hal.buffer_view
+//  CHECK-NEXT:   %[[ARG0_DIM:.+]] = hal.buffer_view.dim<%[[ARG0]] : !hal.buffer_view>[0]
+//  CHECK-NEXT:   %[[ARG0_TENSOR:.+]] = hal.tensor.import %[[ARG0]] "input0" : !hal.buffer_view -> tensor<?xf32>{%[[ARG0_DIM]]}
+//  CHECK-NEXT:   %[[RET_TENSOR:.+]] = util.call @_transientsEntry(%[[ARG0_TENSOR]], %[[STORAGE]])
+//       CHECK:   %[[RET_DIM:.+]] = tensor.dim %[[RET_TENSOR]]
+//       CHECK:   %[[ANNOTATED:.+]] = hal.tensor.transients %[[RET_TENSOR]] : tensor<?xf32>{%[[RET_DIM]]} from %[[STORAGE]] : !hal.buffer
+//  CHECK-NEXT:   %[[RET_VIEW:.+]] = hal.tensor.export %[[ANNOTATED]] "output0" : tensor<?xf32>{%[[RET_DIM]]} -> !hal.buffer_view
+//  CHECK-NEXT:   util.return %[[RET_VIEW]] : !hal.buffer_view
+//  CHECK-NEXT: }
+
+// CHECK-LABEL: util.func private @_transientsEntry
```

Comment:
I am not sure what this generated function is from. I thought initially it is the `_transient_size`, but it is not.